miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/DataTablesResponseDto.java

```diff
@@ -0,0 +1,49 @@
+package uk.ac.bbsrc.tgac.miso.dto;
+
+import java.util.List;
+
+import org.codehaus.jackson.annotate.JsonProperty;
+
+public class DataTablesResponseDto<T> {
```

Comment:
Method-level Javadocs would be nice to clarify what the fields are for without having to look through DataTables' docs (though you could include a link to them in the class-level Javadoc too)